src/modules/sih/sih.hpp

```diff
@@ -113,6 +136,10 @@ class Sih : public ModuleBase<Sih>, public ModuleParams, public px4::ScheduledWo
 	vehicle_global_position_s			_gpos_gt{};
 	uORB::Publication<vehicle_global_position_s>	_gpos_gt_pub{ORB_ID(vehicle_global_position_groundtruth)};
 
+	// airspeed
+	airspeed_s  						_airspeed{};
```

Comment:
This is filled in entirely in `send_airspeed`, can you keep it locally in there?